user-contrib/Ltac2/g_ltac2.mlg

```diff
@@ -73,9 +73,15 @@ let test_lpar_id_rpar =
     lk_kw "(" >> lk_ident >> lk_kw ")"
   end
 
+let check_no_space tok m strm =
+  let n = Stream.count strm in
+  let (_, ep) = Loc.unloc (tok n) in
+  let (bp, _) = Loc.unloc (tok (n + 1)) in
+  if Int.equal ep bp then Some m else None
+
```

Comment:
- It doesn't matter for this application, but shouldn't this function check that all `m` tokens in the lookahead are contiguous, not just the first two?
- Since you've changed the parser API for everyone to implement this functionality, it would be more elegant to provide it for everyone as well, e.g., in `g_prim.mlg`, in lieu of the more limited and similarly named `check_nospace` introduced in #10167 (and change the `bar_cbrace` rule to an entry that uses that).